utilities/gnmi_cli.cc

```diff
@@ -171,6 +171,15 @@ bool extract_interface_node(char **path, char *node_path) {
           snprintf(node_path+strlen(node_path),
                    strlen(FLAGS_device_type_virtual_interface.c_str())+1, "%s",
                    FLAGS_device_type_virtual_interface.c_str());
+          //Validate the device value
+          if((strcmp(value,"virtual-device") != 0) &&
+             (strcmp(value,"physical-device") != 0)) {
+              return -1;
```

Comment:
Will this `virtual-device` and  `physical-device` option be documented somewhere? Or is it already documented and I don't know it exists?

Reply:
Will remove physical device since this is for pci based ports, support for which is being still worked upon. Once this support is available, will surely add the documentation :)